src/components/index.ts

```diff
@@ -0,0 +1,13 @@
+export { default as Avatar } from './Avatar/Avatar';
+export { default as Button } from './Button/Button';
+export { default as Checkbox } from './Checkbox/Checkbox';
+export { default as Datepicker } from './Datepicker/Datepicker';
+export { default as FieldContainer } from './FieldContainer/FieldContainer';
+export { default as FlexLayout } from './FlexLayout/FlexLayout';
+export { default as Form } from './Form/Form';
+export { default as NativeDatepicker } from './NativeDatepicker/NativeDatepicker';
+export { default as Select } from './Select/Select';
+export { default as Tag } from './Tag/Tag';
+export { default as TextArea } from './TextArea/TextArea';
+export { default as TextField } from './TextField/TextField';
+export { default as Toggle } from "./Toggle/Toggle";
```

Comment:
This file needs to be modified to match the new structure. Should export everything on the barrels (index.ts files) and let's also make sure we don't use default exports in the barrels.

Reply:
Soe this file should be something like this:

```
export * from './Tag/Tag';
export * from './TextArea/TextArea';
export * from './TextField/TextField';
export * from "./Toggle/Toggle";
```

Reply:
More like

```
export * from './Tag';
export * from './TextArea';
export * from './TextField';
export * from "./Toggle";
```